aws/ecs/packer/to-upload/peers.sh

```diff
@@ -2,29 +2,221 @@
 # This script print a list of IPs of the EC2 instances in your current Autoscaling Group.
 set -eu
 
-aws=/usr/local/bin/aws
+__instance_tag_key="${1:-WEAVE_ECS}"
```

Comment:
Let's not favor a specific tag. Can you load the tag from a variable in a configuration file? We can use `WEAVE_AWS_PEER_TAG` `/etc/weave/weave.config` . 

The file can be set with User Data just like we do with `SCOPE_AAS_PROBE_TOKE` in `/etc/weave/scope.config` see https://github.com/weaveworks/guides/blob/master/aws-ecs/setup.sh#L127 and https://github.com/weaveworks/integrations/blob/master/aws/ecs/packer/to-upload/run.sh#L26